### PR TITLE
Update manual test case for "Test backing image / Test upload" 

### PR DESCRIPTION
**Pull request by kaxing**
Update backing image test upload manual test case with extended tests for uploading through Rancher UI and Ingress Controller.
To covering following issues:
- longhorn/longhorn#3129
- longhorn/longhorn#2937

### CODE REVIEW

File: docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md [hunk 1]
```diff
@@ -14,6 +14,24 @@ title: Test backing image
 9. Create and attach a volume with the backing image. Verify the data content is correct.
 10. Do cleanup.
 
+### Upload via Rancher URI
+<sup>Related issue: [3129](https://github.com/longhorn/longhorn/issues/3129)</sup>
+1. Create or import cluster into Rancher
```
Comment (khushboo-rancher) [inline docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md:19]: Can you mention the Rancher series here, is it v2.5.x or v2.6.x?

File: docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md [hunk 2]
```diff
@@ -14,6 +14,24 @@ title: Test backing image
...
+2. Go into the cluster
+3. Click Longhorn from the left sidebar
```
Comment (khushboo-rancher) [inline docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md:21]: Are we assuming the Longhorn is already installed? Could you please mention that if possible a version?

File: docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md [hunk 3]
```diff
@@ -14,6 +14,24 @@ title: Test backing image
...
+4. Open Longhorn UI from the Rancher UI
```
Comment (khushboo-rancher) [inline docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md:22]: Can we rephrase this as `Click the Longhorn app in Rancher to navigate to the Longhorn UI`?

File: docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md [hunk 4]
```diff
@@ -14,6 +14,24 @@ title: Test backing image
...
+   - An example URL: <small>`https://rancer.server.domain/k8s/clusters/CLUSTER-ID/api/v1/namespaces/longhorn-system/services/http:longhorn-frontend:80/proxy/#/backingImage`</small>
+5. Create a new backing image with **Upload From Local**
+6. Verify the uploaded image matches its size or checksum
```
Comment (khushboo-rancher) [inline docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md:25]: Do we want to create a volume using this backing image to see if this is working fine?

File: docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md (hunk 3 above)
Comment (kaxing) [reply docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md:22]: I was thinking of the case when install through kubectl, I don't know if this will still be called as an "app", but sure, I can rephrase this section.

File: docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md (hunk 4 above)
Comment (kaxing) [reply docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md:25]: I think it's okay as the attaching part tested in main test case. Validating the checksum should be sufficient for making sure uploading image files behind the proxies.

File: docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md (hunk 1 above)
Comment (kaxing) [reply docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md:19]: Thanks for the reminder. Adding in next commit.

File: docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md (hunk 2 above)
Comment (kaxing) [reply docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md:21]: The entire test case is under `release-specific/v1.2.0/`. I think I can add a few words next to the *related issue*, so the tester will understand it which release they can go back and validate when testing latest version.

**Comment by kaxing** [comment]
> Can we move these test cases to v1.3.0 folder?
Hey @khushboo-rancher only one test is for v1.3.0: to access via Rancher URI.
Please check the changes I made if this will be good enough for the current situation.

File: docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md [hunk 5]
```diff
@@ -14,6 +14,25 @@ title: Test backing image
 9. Create and attach a volume with the backing image. Verify the data content is correct.
 10. Do cleanup.
 
+### Upload via Rancher URI
+<sup>Related issue: [3129](https://github.com/longhorn/longhorn/issues/3129) with fix in Longhorn v1.3.0+</sup>
+1. With Rancher v2.6.x create a cluster and install Longhorn
+   - or import an existing cluster with Longhorn installed
+2. Go into the cluster
+3. Click Longhorn from the Rancher menu, the sidebar on the left
```
Comment (shuo-wu) [inline docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md:22]: I think you mean Longhorn in the cluster explorer mode. It's better to explicitly claim that. Similar for the ingress.

File: docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md [hunk 6]
```diff
@@ -14,6 +14,25 @@ title: Test backing image
...
+4. Click on the Longhorn app to open the UI
+   - An example URL: <small>`https://rancer.server.domain/k8s/clusters/CLUSTER-ID/api/v1/namespaces/longhorn-system/services/http:longhorn-frontend:80/proxy/#/backingImage`</small>
+5. Create a new backing image with **Upload From Local**
```
Comment (shuo-wu) [inline docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md:25]: I would recommend testing with a large backing image (> 1Gi).

**Review by innobead — DISMISSED** [review]
LGTM, but please resolve the comments from others.

note: From 2.6, the default UI is actually cluster explorer already but explicitly mentioning it is OK.

File: docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md (hunk 6 above)
Comment (kaxing) [reply docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md:25]: These two cases were extended from the main test, already mentioned using a large image file. I think it will be okay to keep these two sub-cases minimal.

File: docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md (hunk 2 above)
Comment (kaxing) [reply docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md]: @khushboo-rancher I will resolve this by now. As I've put Longhorn version next to the related issue, should be good hint for us to relocate these test cases in the future.

File: docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md (hunk 5 above)
Comment (kaxing) [reply docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md]: I've updated the document to use the official wording: _Cluster Explorer_ and _Apps & Marketplace_ from Rancher v2.6.x
